receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,374 @@
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [disk_state]
+  vcenter.datastore.disk.utilization:
+    enabled: true
+    description: The utilization of the datastore.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the host system.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.usage:
+    enabled: true
+    description: The amount of CPU in Hz used by the host.
+    unit: "MHz"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.throughput:
+    enabled: true
+    description: The throughput to the host system's disk.
+    unit: By/s
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.latency.avg:
+    enabled: true
+    description: The latency of operations to the host system's disk.
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [latency_direction, latency_type]
+    extended_documentation: This latency is the sum of the device and kernel write latencies. Requires Performance Counter level 2 for metric to populate.
+  vcenter.host.disk.latency.total:
+    enabled: true
+    description: The total reported total latency (device and kernel times)
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [latency_direction]
+  vcenter.host.memory.utilization:
+    enabled: true
+    description: The percentage of the host system's memory capacity that is being utilized.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.memory.usage:
+    enabled: true
+    description: The amount of memory the host system is using.
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.network.throughput:
+    enabled: true
+    description: The amount of data that was sent or received over the network by the host.
+    unit: "{KBy/s}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [throughput_direction]
+  vcenter.host.network.usage:
+    enabled: true
+    description: The sum of the data transmitted and received for all the NIC instances of the host.
+    unit: "{KBy/s}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.network.errors:
+    enabled: true
+    description: The summation of errors on the host network.
+    unit: "{errors}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [throughput_direction]
+  vcenter.host.network.packets:
+    enabled: true
+    description: The number of packets sent over an interval.
+    unit: "{packets/sec}"
```

Comment:
Any chance there's a counter version of this?

Reply:
I could not find one that didn't use the performance manager API which uses an interval based approach. If you find a doc that references it I'd be happy to look but I could only find the one.